2023-01-03_infSILAC_Processing_and_QC_of_SILAC_data.Rmd

```diff
@@ -210,7 +171,7 @@ missing_tallies <- ratios %>%
   group_by(Replicate, missing) %>%
   tally()
 
-missing_tallies$Replicate <- factor(missing_tallies$Replicate, levels=c("1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16"))
+missing_tallies$Replicate <- factor(missing_tallies$Replicate, levels=replicates)
```

Comment:
You've already defined a vector called `replicates` with values 1-16, so can just re-use that!